### ISSUE TITLE
REST API doesn't accept prettified JSON containing tabs

### ISSUE THREAD

**Issue by philenius** [floragunncom/search-guard#634]
I had some difficulties when dealing with the REST API of Search Guard. I was trying to create an internal user.

My HTTP request:
```bash
curl -u admin:admin -H "Content-Type: application/json" -XPUT https://my-es-cluster.domain:9200/_searchguard/api/internalusers/kirk -d'
{
	"password": "kirk",
	"roles": ["captains", "starfleet"],
	"attributes": {
    	"attribute1": "value1",
		"attribute2": "value2",
	}
}'
```

If I execute this request with curl, I'll get a HTTP response code 400. I reconstructed the same request with Postman and still got the same response:
```
HTTP 400 Bad Request
{
	"status": "error",
	"reason": "Could not parse content of request."
}
``` 

Then, I found out that replacing all tabs in the JSON body by space characters returns a HTTP status code 200. It looks like Search Guard seems to be a little bit picky when it comes to JSON parsing. Using minified JSON without any whitespaces works fine, too.

Search Guard deployment environment:
* Search Guard: 6.5.4-33.0
* Elasticsearch: 6.5.4-1
* OS: RHEL 7.4 (Maipo)
* JVM: OpenJDK 1.8.0_161-b14
* Number of nodes: 2 (test cluster)

**Comment by floragunncom** [comment]
Please split your issues in one issue per github issue

**Comment by philenius** [comment]
Thanks for your hint. Done!

**Comment by floragunncom** [comment]
Fixed with v24.2 released today.
